src/app/rundown-overview/components/rundown-overview/rundown-overview.component.html

```diff
@@ -1,15 +1,15 @@
 <h1 class="c-rundown-overview__title">Rundowns</h1>
 
-<div class="c-rundown-overview__loading-rundowns" *ngIf="isLoading">
+<div *ngIf="isLoading" class="c-rundown-overview__loading-rundowns">
   <mat-card-title>Loading rundowns...</mat-card-title>
 </div>
 
-<div class="c-rundown-overview__no-rundowns" *ngIf="!isLoading && basicRundowns.length === 0">
+<div *ngIf="!isLoading && basicRundowns.length === 0" class="c-rundown-overview__no-rundowns">
   <h2>There are currently no rundowns available.</h2>
   <p>Check your connection and if you have configured the Ingest Gateway.</p>
 </div>
 
-<table mat-table [dataSource]="basicRundowns" *ngIf="!isLoading && basicRundowns.length > 0">
+<table *ngIf="!isLoading && basicRundowns.length > 0" mat-table [dataSource]="basicRundowns">
```

Comment:
Why?

Reply:
It's an old "design pattern" that was an easy way to display data before the advent of modern standards-based browsers and is now considered a bad practice in a lot of cases since they are more difficult to optimize for the requirements of a modern webpage (responsiveness, accessibility, SEO etc - not that we care for most of those things...)